inference-engine/src/gna_plugin/gna_graph_tools.hpp

```diff
@@ -711,4 +711,22 @@ inline void CNNNetworkRemoveLayer(CNNLayerPtr layer, bool checkDims = true) {
 
     // removing layer->osp, and layer->isp connection not necessary - layer will delete it by itself
 }
+
+/**
+ * @brief returns dimensions order for permute from input to output layout
+ * @param in_layout
+ * @param out_layout
+ */
+inline std::vector<int> GetPermuteOrder(Layout in_layout, Layout out_layout) {
```

Comment:
i would move this into either PermuteSequence class directly or at least into gna_permute.hpp